src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohort.java

```diff
@@ -52,13 +47,15 @@
     @Argument(
             fullName = "cohort-extract-table",
             doc = "Fully qualified name of the table where the cohort data exists (already subsetted)",
-            optional = false
+            mutex={"cohort-avro-file-name"},
+            optional = true
     )
     private String cohortTable = null;
 
     @Argument(
             fullName = "cohort-avro-file-name",
             doc = "Path of the cohort avro file",
+            mutex={"cohort-extract-table"},
```

Comment:
i'm so sorry to be this person but can you put spaces before and after the `=` on all these mutex sets so it's consistent?